OWML.SampleMods/OWML.LoadCustomAssets/LoadCustomAssets.cs

```diff
@@ -48,6 +54,10 @@ public override void Configure(IModConfig config)
             var speed = config.GetSettingsValue<float>("speed");
             var power = config.GetSettingsValue<float>("power");
             var enableSuperMode = config.GetSettingsValue<bool>("enableSuperMode");
+            var selectedEnum = config.GetSettingsValue<ABC>("thing");
+            var selectedString = config.GetSettingsValue<string>("thing");
```

Comment:
seems a bit excessive to support both string and enum at the same time?

Reply:
String is it's native format, no reason to not support it 🤷‍♂️

Reply:
If we were to remove one we'd remove Enum, but I think both can be useful (if you're feeling lazy use string / int, if you want cleaner code with less magic strings use Enum).

Reply:
right, the enum is the special case. covered by other comments 👍